src/LanguageServer/Protocol/Handler/CodeActions/CodeActionHelpers.cs

```diff
@@ -419,18 +418,13 @@ private static CodeActionKind GetCodeActionKindFromSuggestedActionCategoryName(s
             _ => throw ExceptionUtilities.UnexpectedValue(categoryName)
         };
 
-    private static CodeActionKind GetRefactoringKind(IUnifiedSuggestedAction suggestedAction)
-    {
-        if (suggestedAction is not ICodeRefactoringSuggestedAction refactoringAction)
```

Comment:
the entire itnerface impl existed to just access a single Kind property.  Simpler to just store that as data on the unified action.